### PR TITLE
add content on write transactions and update images

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-bloom-41.surge.sh

File: modules/ROOT/pages/bloom-tutorial/scene-actions.adoc [hunk 1]
```diff
@@ -7,6 +7,15 @@ _Scene actions_ are parameterized Cypher queries, much like xref:/bloom-visual-t
 The difference is that in a Scene action, the parameters can be the selected elements in your current selection instead of all available and matching elements in your graph.
 Also, whereas Scene actions are defined in the Perspective drawer under _Saved Cypher_, they are invoked from the context menu.
 
+[[write-transaction-action]]
+[IMPORTANT]
+====
+A Scene action can be sent as either a READ or a WRITE transaction to the database (provided you have appropriate permissions).
+In order to allow a WRITE transaction, you need to first enable this in the xref:bloom-visual-tour/settings-drawer.adoc[**Settings**] drawer.
```
Comment (joknelid) [inline modules/ROOT/pages/bloom-tutorial/scene-actions.adoc:14]: I had maybe replaced allow with enable or send here, allow sounds more like access control which is not really what this is about.

File: modules/ROOT/pages/bloom-tutorial/search-phrases-advanced.adoc [hunk 2]
```diff
@@ -111,3 +112,11 @@ MATCH p = ()-[r:CONNECTED_TO]->() RETURN p
 
 * Furthermore, be aware that it is possible to modify data with a Search phrase as any valid Cypher query can be used.
   It is not recommended to use Search phrase for this goal as an end-user might not be aware of the consequences of running a Search phrase that includes WRITE transactions.
+
+*  The Cypher query in the Search phrase can be sent as either a READ or a WRITE transaction to the database (provided you have appropriate permissions).
+In order to allow a WRITE transaction, you need to first enable this in the xref:bloom-visual-tour/settings-drawer.adoc[**Settings**] drawer.
```
Comment (joknelid) [inline modules/ROOT/pages/bloom-tutorial/search-phrases-advanced.adoc:117]: Same as pervious comment.

**Review by joknelid — APPROVED** [review]
Everything is correct and clear so approving the PR. One suggestion for improvement!

**Review by JeffG-N4j — APPROVED** [review]

File: modules/ROOT/pages/bloom-tutorial/scene-actions.adoc (hunk 1 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/bloom-tutorial/scene-actions.adoc]: Point taken! Thank you!

File: modules/ROOT/pages/bloom-tutorial/search-phrases-advanced.adoc (hunk 2 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/bloom-tutorial/search-phrases-advanced.adoc]: 👍

**Pull request merged**